src/app/posts/posts.component.html

```diff
@@ -0,0 +1,43 @@
+<div style="margin-top: 6%; margin-bottom: 1px">
```

Comment:
inline styling should be avoided everywhere

Reply:
done